war/src/main/js/widgets/add/addform.less

```diff
@@ -23,14 +23,14 @@
 
     .btn-decorator {
       position: absolute;
-      bottom: -1px;
-      left: -1px;
+      bottom: 0px;
       padding: 20px 30px;
       width: auto;
       border-top-right-radius: 10px;
       border: 1px solid #aaccbb;
```

Comment:
Me too but we are using the this design in job configuration. I can push another PR to change both.